src/App/Documentation/components/Panel/__snapshots__/index.test.js.snap

```diff
@@ -16,19 +16,55 @@ exports[`Documentation: Panel Overview renders 1`] = `
     showCasePanel={true}
   >
     <Panel
-      bodyContent={
-        Array [
-          "Your main panel content is put here.",
-          "Some more content over here.",
-          "And one more line, just to be safe.",
-        ]
-      }
       footerBtnText="Footer button"
       footerText="Footer content goes here."
       subTitle="Panel subtitle"
       title="Panel title"
-    />
+    >
+      <p>
+        Your main panel content is put here.
+      </p>
+      <p>
+        Some more content over here.
+      </p>
+      <p>
+        And one more line, just to be safe.
+      </p>
+    </Panel>
   </ComponentPreview>
+  <Alert
+    icon="warning"
+    type="warning"
+  >
+    <h4>
+      Deprecation warning
+    </h4>
+    <p>
+      Only using 
+      <PrismCode
+        className="language-html"
+        component="code"
+      >
+        &lt;header&gt;
+      </PrismCode>
+       and 
+      <PrismCode
+        className="language-html"
+        component="code"
+      >
+        &lt;footer&gt;
+      </PrismCode>
+       is deprecated. Use our 
+      <Property
+        value=".panel-header"
+      />
+       and 
+      <Property
+        value=".panel-footer"
+      />
+       classes instead.
```

Comment:
Details, but don't think "instead" is the best wording here, considering we don't remove anything.